### PR TITLE
updated test cases and api codes

### PR DESCRIPTION
**Pull request by Rupeshiya**
---
name:  Updated failing test cases
about: updated test cases and API codes
title: ''Test cases"
labels: ''bug"
assignees: ''Rupeshiya"

---

### **Problem**
Github Issue Number: #118 
The test cases for comment and post, events were failingl_

### **Solution of problem**
Written test cases to fix them

### **In Case of UI Changes**
No changes required

### **Type of Change**
[&check;] Bug fix
[ ] New Feature
[ ] Development of UI/UX prototypes
[&check;] Small refactor
[ ] Change in Documentation

### **Checklist**
[&check;] My code follows the same style as the codebase
[ ] My Code change requires a change in documentation
[ ] I have updated the Readme accordingly
[&check;] I made PR against **development branch**
[&check;] I have run the test cases locally and it's passing.
[&check;] I have squashed my commits

### CODE REVIEW

**Comment by Rupeshiya** [comment]
@devesh-verma Please review!!

File: app/routes/comment.js [hunk 1]
```diff
@@ -3,37 +3,43 @@ const router = express.Router()
 const auth = require('../middleware/auth')
 const commentController = require('../controllers/comment')
 
+// CREATE COMMENT
 router.post(
-  '/',
+  '/:id',
```
Comment (devesh-verma) [inline app/routes/comment.js:8]: @Rupeshiya how can you get the ID before even creating the comment

File: app/controllers/comment.js [hunk 2]
```diff
@@ -87,24 +99,26 @@ module.exports = {
       // CHECKS IF THE USER HAS ALREADY UPVOTED THE COMMENT
       comment.votes.upVotes.user.filter(user => {
         if (JSON.stringify(user) === JSON.stringify(userId)) {
-          return res.status(HttpStatus.BAD_REQUEST).json({ error: 'Bad request' })
+          return res.status(HttpStatus.BAD_REQUEST).json({
+            error: 'Bad request'
+          })
         }
       })
       // CHECKS IF THE USER HAS ALREADY DOWNVOTED THE COMMENT
```
Comment (devesh-verma) [inline app/controllers/comment.js:107]: we can remove this entire logic as we dont have downvote in our platform.

File: app/routes/comment.js (hunk 1 above)
Comment (Rupeshiya) [reply app/routes/comment.js:8]: @devesh-verma It is a POST ID on which comment is going to be added!

File: app/controllers/comment.js (hunk 2 above)
Comment (Rupeshiya) [reply app/controllers/comment.js:107]: @devesh-verma I talked to @jaskirat2000 He told me that we will downvote on comments but not on posts.

**Comment by jaskirat2000** [comment]
✌

**Pull request merged**